maro/utils/logger.py

```diff
@@ -87,72 +87,47 @@ class Logger(object):
     """A simple wrapper for logging.
 
     The Logger hosts a file handler and a stdout handler. The file handler is set
-    to ``DEBUG`` level and will dump all the logging info to the given ``dump_folder``.
-    The logging level of the stdout handler is decided by the ``stdout_level``,
-    and can be redirected by setting the environment variable ``LOG_LEVEL``.
-    Supported ``LOG_LEVEL`` includes: ``DEBUG``, ``INFO``, ``WARN``, ``ERROR``,
-    ``CRITICAL``, ``PROCESS``.
-
-    Example:
-        ``$ export LOG_LEVEL=INFO``
+    to ``DEBUG`` level and will dump all logs info to the given ``dump_path``.
+    Supported log levels include: ``DEBUG``, ``INFO``, ``WARN``, ``ERROR``, ``CRITICAL``, ``PROCESS``.
 
     Args:
         tag (str): Log tag for stream and file output.
         format_ (LogFormat): Predefined formatter. Defaults to ``LogFormat.full``.
-        dump_folder (str): Log dumped folder. Defaults to the current folder.
-            The dumped log level is ``logging.DEBUG``. The full path of the
-            dumped log file is `dump_folder/tag.log`.
-        dump_mode (str): Write log file mode. Defaults to ``w``. Use ``a`` to
-            append log.
-        extension_name (str): Final dumped file extension name. Defaults to `log`.
-        auto_timestamp (bool): Add a timestamp to the dumped log file name or not.
-            E.g: `tag.1574953673.137387.log`.
-        stdout_level (str): the logging level of the stdout handler. Defaults to
-            ``DEBUG``.
+        dump_path (str): Path of file for dumping logs. Must be an absolute path. The log level for dumping is
+            ``logging.DEBUG``. Defaults to None, in which case logs generated by the logger will not be dumped
+            to a file.
+        dump_mode (str): Write log file mode. Defaults to ``w``. Use ``a`` to append log.
+        stdout_level (str): the logging level of the stdout handler. Defaults to ``INFO``.
+        file_level (str): the logging level of the file handler. Defaults to ``DEBUG``.
     """
 
     def __init__(
-        self, tag: str, format_: LogFormat = LogFormat.simple, dump_folder: str = cwd, dump_mode: str = 'w',
```

Comment:
1. I think It's more straightforward to let the user directly specify the log file path (we have a ``log_path`` field in the config file for this). Instead of implicitly joining the folder path, the tag, an optional timestamp and an optional extension in the code like the old version did, why not just let the user manage the log path? I don't see how the timestamp makes log management any easier.
 2. Adding a file level gives the user the option to specify the type of log messages the user wants to see dumped in a file. Again, we have fields in the config file template for this.